src/work-allocation-2/containers/work-case-list/work-case-list.component.html

```diff
@@ -0,0 +1,98 @@
+<div *ngIf="pagination; else tableSummary" class="pagination-top" attr.aria-label="{{ casesTotal }} results have been found" role="status">
+  <span class="text-16" id="search-result-summary__text">Displaying
+      <span class="govuk-!-font-weight-bold">{{ getFirstResult() }}</span>
```

Comment:
_Nitpick_:
This block is indented too far.

Also, this looks identical to a block I commented on above. Can this be a common component rather than duplicated in several places?

Reply:
I've formatted the entire html file now.